src/integrations/__tests__/unixIntegrationManager.spec.ts

```diff
@@ -57,25 +57,26 @@ describeUnix('UnixIntegrationManager', () => {
 
   test('.enforce() should create dirs and symlinks properly', async() => {
     await integrationManager.enforce();
-    expect(fs.promises.readdir(integrationDir)).resolves.not.toThrow();
     for (const name of await fs.promises.readdir(resourcesDir)) {
       const integrationPath = path.join(integrationDir, name);
+      const expectedValue = path.join(resourcesDir, name);
 
-      expect(fs.promises.readlink(integrationPath, 'utf8')).resolves.not.toThrow();
+      await expect(fs.promises.readlink(integrationPath, 'utf8')).resolves.toEqual(expectedValue);
     }
     for (const name of await integrationManager.getDockerCliPluginNames()) {
       const pluginPath = path.join(dockerCliPluginDir, name);
+      const expectedValue = path.join(integrationDir, name);
 
-      expect(fs.promises.readlink(pluginPath, 'utf8')).resolves.not.toThrow();
+      await expect(fs.promises.readlink(pluginPath, 'utf8')).resolves.toEqual(expectedValue);
     }
   });
 
   test('.remove() should remove symlinks and dirs properly', async() => {
     await createTestSymlinks(resourcesDir, integrationDir, dockerCliPluginDir);
 
     await integrationManager.remove();
-    expect(fs.promises.readdir(integrationDir)).rejects.toThrow();
-    expect(fs.promises.readdir(dockerCliPluginDir)).resolves.toEqual([]);
+    await expect(fs.promises.readdir(integrationDir)).rejects.toThrow();
```

Comment:
Can you specify an error that this `readdir` will throw on both macOS and linux?